scripts/main.zeek

```diff
@@ -26,52 +29,49 @@ export {
 
 	## A default logging policy hook for the stream.
 	global log_policy: Log::PolicyHook;
+
+	## Indicator of a request related to GOZI
+	redef enum HTTP::Tags += { URL_GOZIMALWARE, };
 }
 
 # Regex - make them globals so they are compiled only once!
 global rar_regex = /.*\/(stilak|cook|vnc)(32|64)\.rar$/i;
 global b64_regex = /^\/[^[:blank:]]+\/([a-zA-Z0-9\/]|_\/?2\/?F|_\/?2\/?B|_\/?0\/?A|_\/?0\/?D){200,}\.[a-zA-Z0-9]+$/;
 
-redef record connection += {
-	gozi: Info &optional;
-};
-
-# Initialize logging state.
-hook set_session(c: connection)
-	{
-	if ( c?$gozi )
-		return;
-
-	c$gozi = Info($ts=network_time(), $uid=c$uid, $id=c$id);
-	}
-
-function log_gozi_detected(c: connection)
+function log_gozi_detected(c: connection, http_method: string, payload: string,
+    enable_detailed_logs: bool)
```

Comment:
Is it necessary to pass in `enabled_detailed_logs` since it's scope is the package?

Reply:
Fixed